sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -402,12 +408,63 @@ public synchronized NetworkName getNetworkName() {
     }
 
     public synchronized Client setMaxAttempts(int maxAttempts) {
+        if (maxAttempts <= 0) {
+            throw new IllegalArgumentException("maxAttempts must be greater than zero");
+        }
         this.maxAttempts = maxAttempts;
         return this;
     }
 
     public synchronized int getMaxAttempts() {
-        return maxAttempts != null ? maxAttempts : DEFAULT_MAX_ATTEMPTS;
+        return maxAttempts;
+    }
+
+    /**
+     * @return maxBackoff The maximum amount of time to wait between retries
+     */
+    public Duration getMaxBackoff() {
+        return maxBackoff;
+    }
+
+    /**
+     * The maximum amount of time to wait between retries. Every retry attempt will increase the wait time exponentially
+     * until it reaches this time.
+     *
+     * @param maxBackoff The maximum amount of time to wait between retries
+     * @return {@code this}
+     */
+    public Client setMaxBackoff(Duration maxBackoff) {
```

Comment:
We can avoid `synchronized` since we're using `volatile` variables. This is safe to do as long as the variable doesn't have a read followed by a write in the same thread (e.g. incrementing, etc).